parsers/US_CA.py

```diff
@@ -0,0 +1,79 @@
+# The arrow library is used to handle datetimes
+import arrow
+# The request library is used to fetch content through HTTP
+import requests
+
+from datetime import datetime, timedelta
+
+
+def fetch_production(country_code='FR', session=None):
+    """Requests the last known production mix (in MW) of a given country
+
+    Arguments:
+    country_code (optional) -- used in case a parser is able to fetch multiple countries
+    session (optional)      -- request session passed in order to re-use an existing session
+
+    Return:
+    A dictionary in the form:
+    {
+      'countryCode': 'FR',
+      'datetime': '2017-01-01T00:00:00Z',
+      'production': {
+          'biomass': 0.0,
+          'coal': 0.0,
+          'gas': 0.0,
+          'hydro': 0.0,
+          'nuclear': null,
+          'oil': 0.0,
+          'solar': 0.0,
+          'wind': 0.0,
+          'geothermal': 0.0,
+          'unknown': 0.0
+      },
+      'storage': {
+          'hydro': -10.0,
+      },
+      'source': 'mysource.com'
+    }
+    """
+    # caiso.com provides daily data until the day before today
+    s = requests.session()
+    yesterday = arrow.utcnow().to('US/Pacific').shift(days=-1)
+    url = 'http://content.caiso.com/green/renewrpt/' + yesterday.format('YYYYMMDD')  +'_DailyRenewablesWatch.txt'
+
+    response = s.get(url)
+    obj = response.text.splitlines()
+
+    dailyData = []
+
+    for i in range(0, 24):
+        data = {
+            'countryCode': country_code,
+            'production': {},
+            'storage': {},
+            'source': 'content.caiso.com',
+        }
+        data['countryCode'] = 'US_CA'
+        data['datetime'] = datetime.now()
+        data['production'] = {}
+        data['production']['biomass'] = int(obj[i+2].split('\t')[5])
```

Comment:
if we stick with getting the data like this it'll need to be float rather than int